### PR TITLE
Added Distance /controllers/index

### PR DESCRIPTION
**Pull request by agbolton**
The distance can be called after getting an array of trails. Trying this simple pull request first. WIll add the other story/subtasks separately. 

Still need to how we are calling these from the front end.

### CODE REVIEW

File: src/controllers/distance.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export function findDistanceToTrail(trail, coordinates) {
+  const distance = latLonDistance(trail.latitude, trail.longitude, coordinates);
+  return distance;
+}
+
+function latLonDistance(trailLat, trailLon, coordinate) {
```
Comment (mgarber93) [inline src/controllers/distance.js:6]: IMO this should have taken two coordinates (start and end?) as parameters. As far as the math goes I'm pretty lost, maybe link an article or something in a comment?

File: src/controllers/distance.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export function findDistanceToTrail(trail, coordinates) {
```
Comment (mgarber93) [inline src/controllers/distance.js:1]: I don't think this needs to exist.

File: src/controllers/distance.js (hunk 1 above)
Comment (mgarber93) [reply src/controllers/distance.js:6]: Maybe use [Polar coordinate flat-Earth formula](https://en.wikipedia.org/wiki/Geographical_distance) with a separate function to convert to colatitude in radians?

Comment (agbolton) [reply src/controllers/distance.js:6]: Yeah just took it straight from here - https://www.geodatasource.com/developers/javascript

Comment (agbolton) [reply src/controllers/distance.js:6]: Also, we definitely can refactor into both being objects and then just referencing .lat / .lon

**Review by mgarber93 — APPROVED** [review]

**Pull request merged**